mgcpy/mgc/tests/dcorr_test.py

```diff
@@ -0,0 +1,12 @@
+import pytest
```

Comment:
Move the tests into a tests folder under your newly created module under `mgcpy`.